client/src/Header.js

```diff
@@ -1,27 +1,96 @@
 import Logo from "./assets/images/CTY-logo-rectangle.png";
 import { NavLink } from "react-router-dom";
+import { useState, useEffect } from "react";
 import "./Header.css";
 
 const Header = () => {
+	const [role, setRole] = useState(null);
+
+	useEffect(() => {
+		const userType = localStorage.getItem("userType");
+		setRole(userType);
+	}, []);
+
 	return (
 		<header className="header">
 			<img className="logo" src={Logo} alt="Code Your Future logo" />
 			<nav className="nav-list">
-				<li>
-					<NavLink exact to="/" className="nav-link" activeClassName="active">
-						Home{" "}
-					</NavLink>
-				</li>
-				<li>
-					<NavLink
-						exact
-						to="/list-tenders"
-						className="nav-link"
-						activeClassName="active"
-					>
-						All Tenders{" "}
-					</NavLink>
-				</li>
+				{role === "admin" && (
+					<>
+						<NavLink
+							exact
+							to="/list-tenders"
+							className="nav-link"
+							activeClassName="active"
+						>
+							All Tenders
+						</NavLink>
+						<>
```

Comment:
Why do we have an extra level of indentation in there an in other places?

Reply:
I have removed it.